input/posts/cs/split_and_display_channels_of_color_image.md

```diff
@@ -0,0 +1,22 @@
+﻿title: Jednotlivé barevné kanály
+lead: Rozdělení a zobrazení barevných kanálů obrázku
+Published: 2019-01-03
+Tags: [matlab, RGB, channels]
+datazooFiles: imgs/kytka256.jpg
+prerequisites: [Matlab]
+Authors: [tesar-tech, Karina9510]
+---
+Script zobrazuje jednotlivé barevné kanály na RGB obrázku.
```

Comment:
Celý ten popis je hrozně zmatený. Zkus to jednoduše, nehledej v tom složité věci...